django/contrib/admin/views/main.py

```diff
@@ -35,6 +36,16 @@
     ALL_VAR, ORDER_VAR, ORDER_TYPE_VAR, SEARCH_VAR, IS_POPUP_VAR, TO_FIELD_VAR)
 
 
+class ChangeListForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        super(ChangeListForm, self).__init__(*args, **kwargs)
+        self.fields = {
```

Comment:
Is there a reason these fields aren't declared on the class?

Reply:
sorry, I couldn't understand this.

Reply:
Oh, it's because `SEARCH_VAR`, etc. are variables.

Reply:
@timgraham I confirm this was my intent to make sure that constants are used as fieldnames.